romancal/datamodels/reference_files/gain.py

```diff
@@ -0,0 +1,12 @@
+from .referencefile import ReferenceFileModel
+
+class GainModel(ReferenceFileModel):
+    """
+    A data model for 2D gain.
+
+    Parameters
+    __________
+    data : numpy float32 array
+         The gain
```

Comment:
This should also work with file names. And it does because the implementation is in stadatamodels but the `Parameters` signature is confusing. Also, it's a larger conversation for the team but if reference files are allowed to be initialized with a data array should the constructor also accept the `meta` attributes?

Reply:
I am a little confused - data in the gain schema is the values of the gain map. I don't understand how that is interchangeable with filenames?

Reply:
In case a `GainModel` is initialized with a file, something that the pipeline does all the time, the call will be:

```
model = GainModel(filename)
```
However, for testing purposes and also when generating reference files, it may be useful to initialize the model with a data array, which is the case I thought is currently covered in `Parameters`:
```
model = GainModel(data)
model.meta.pedigree = "Ground"
...
```
All I was saying is that while both use cases are covered already by stdatamodels, the Parameters section does not reflect this.